src/LaunchDarkly.ClientSdk/LdClient.cs

```diff
@@ -244,16 +281,19 @@ async Task StartAsync()
         /// </summary>
         /// <remarks>
         /// <para>
-        /// In offline mode, this constructor will return immediately. Otherwise, it will wait and block on
-        /// the current thread until initialization and the first response from the LaunchDarkly service is
-        /// returned, up to the specified timeout. If the timeout elapses, the returned instance will have
-        /// an <see cref="Initialized"/> property of <see langword="false"/>.
+        /// The constructor will return the <see cref="LdClient"/> instance once the first response from
+        /// the LaunchDarkly service is returned, or immediately if offline, or when the the specified
+        /// wait time elapses.  If the max wait time elapses, the returned instance will have
+        /// an <see cref="Initialized"/> property of <see langword="false"/>, but the instance will continue
+        /// trying to get fresh feature flags.
```

Comment:
For reviewers: Documentation changes are just tidying up the contents and making each of the init docs more consistent with each other.  They diverged in subtle ways over time.